tutorials/monitoring/tutorial.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitoring NServiceBus solutions
+reviewed: 2017-10-02
+summary: Getting started with the monitoring capabilities built in to the Particular Service.
+---
+
+This tutorial will guide you through getting started with the monitoring capabilities built in to the Particular Service Platform.
+
+You can follow this tutorial to configure any NServiceBus based solution for monitoring with the Particular Service Platform. If you do not have a solution to monitor just yet, you can get started by downloading our sample solution.
+
+downloadbutton
+
+
+## Component overview
+
+The following diagram shows all the components involved in monitoring an NServiceBus solution with the Particular Service Platform.
+
+```mermaid
+graph TD
+
+subgraph Endpoint1
+  metrics1(NServiceBus.Metrics) --> plugin1
+  plugin1(NServiceBus.Metrics.ServiceControl)
+end
+
+subgraph Endpoint2
+  metrics2(NServiceBus.Metrics) --> plugin2
+  plugin2(NServiceBus.Metrics.ServiceControl) 
+end
+
+plugin1 --> q(Monitoring Queue)
+plugin2 --> q(Monitoring Queue)
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+```
+
+![the above renders like this](component-overview.png)
+
+Each endpoint in the solution is configured to collect metrics about it's own operation using the NServiceBus.Metrics NuGet package. This package hooks into the NServiceBus pipeline to calculate and expose key metric data. It also adds headers to outgoing messages to track queue length.
+
+The NServiceBus.Metrics.ServiceControl NuGet package collects the generated metric data in memory and periodically sends it to a centralized monitoring queue. It does this using the same transport that the endpoints use to exchange business messages.
```

Comment:
![image](https://user-images.githubusercontent.com/677704/31216197-0dbd001c-a9b3-11e7-8fb8-1d373482b407.png)

While that makes it more explicit, as a reader I may still be asking questions like:

> What if the endpoint goes down and loses the data before it is sent? Sounds a bit sketchy to me...

Do we need to preempt those types of questions anywhere?